BootloaderCorePkg/Stage2/Stage2.c

```diff
@@ -34,6 +39,20 @@ LoadComponentCallback (
   default:
     break;
   }
+
+  if(CbInfo)

```

Comment:
I think you should define a new function for this.  And only call this with  PROGESS_ID_AUTHENTICATE case.

Reply:
done